src/backtrace/dbghelp.rs

```diff
@@ -93,6 +93,31 @@ pub unsafe fn trace(cb: &mut FnMut(&super::Frame) -> bool) {
         Err(()) => return, // oh well...
     };
 
+    // On x86_64 and ARM64 we opt to not use the default `Sym*` functions from
+    // dbghelp for getting the function table and module base. Instead we use
+    // the `RtlLookupFunctionEntry` function in kernel32 which will account for
+    // JIT compiler frames as well. These should be equivalent, but using
+    // `Rtl*` allows us to backtrace through JIT frames.
```

Comment:
Just thinking that we should perhaps mention that this is limited to in-process walking only, which is fine for `backtrace`'s use case.